addons/web/static/src/search/control_panel/control_panel.js

```diff
@@ -46,12 +47,25 @@ export class ControlPanel extends Component {
 
         const { viewSwitcherEntries, viewType } = this.env.config;
         for (const view of viewSwitcherEntries || []) {
-            useCommand(_t("Show %s view", view.name), () => this.onViewClicked(view.type), {
+            useCommand(_t("Show %s view", view.name), () => this.switchView(view.type), {
                 category: "view_switcher",
                 isAvailable: () => view.type !== viewType,
             });
         }
 
+        if (viewSwitcherEntries?.length > 1) {
```

Comment:
necessary because, as I was adding it by mistake in views with CP but no view switcher visible, like a form view.